src/features/island/plots/components/FertilePlot.tsx

```diff
@@ -34,10 +52,29 @@ const FertilePlotComponent: React.FC<Props> = ({
 }) => {
   const [showTimerPopover, setShowTimerPopover] = useState(false);
 
-  const harvestSeconds = cropName ? CROPS()[cropName].harvestSeconds : 0;
-  const timeLeft = plantedAt ? getTimeLeft(plantedAt, harvestSeconds) : 0;
+  const [_, setRender] = useState<number>(0);
+
+  let harvestSeconds = cropName ? CROPS()[cropName].harvestSeconds : 0;
+  const readyAt = plantedAt ? plantedAt + harvestSeconds * 1000 : 0;
+
+  let startAt = plantedAt ?? 0;
+  if (cropName && bumpkin) {
+    const fertiliserName = fertiliser?.name ?? undefined;
+    harvestSeconds = getCropTime({
+      crop: cropName,
+      inventory,
+      collectibles,
+      bumpkin,
+      buds: buds ?? {},
+      plot,
+      fertiliser: fertiliserName,
+    });
+    startAt = readyAt - harvestSeconds * 1000;
+  }
+  const timeLeft = readyAt > 0 ? (readyAt - Date.now()) / 1000 : 0;
   const isGrowing = timeLeft > 0;
 
+  // REVIEW: Is this still needed after changing to LiveProgressBar?
   useUiRefresher({ active: isGrowing });
```

Comment:
Looks like this is still needed for now - the progress bar updates, however the crops do not change their growth icon. (e.g. Seedling > Plant).